about.html

```diff
@@ -96,15 +102,7 @@ <h4 class="about-title-details">Lorem ipsum dolor sit amet consectetur, adipisic
         </ul>
       </div>
     </section>
-    <section class="footerLast">
-      <div class="footer-logo">
-        <img src="image/footer-img.png" alt="">
-      </div>
-      <div class="footer-description">
-        <h5 class="footer-title">2015 Creative Commons Korea.</h5>
-        <h6 class="footer-title2">Lorem ipsum dolor sit, amet</h6>
-      </div>
-    </section>
+   
   </section>
 
       <script src="script.js"></script>
```

Comment:
- Kindly, you forget to add the footer for your about page on both desktop and mobile versions, please add it to have the exact design as the given one
![2023-03-25_09-16](https://user-images.githubusercontent.com/52010789/227701050-e4a341fd-7449-4e67-acbb-29884cb8b9cd.png)